Scripts/script-JoinIfSingleElementOnly.yml

```diff
@@ -0,0 +1,28 @@
+commonfields:
+  id: JoinIfSingleElementOnly
+  version: -1
+name: JoinIfSingleElementOnly
```

Comment:
"Join" has the connotation of joining list with a separator (python `",".join(somelist)` / js `somelist.join(",")`

maybe we should call it `ExtractSingleElementFromList` / `GetSingleElementFromList` .. wdyt?

Reply:
neither, because when it's not single element, it stays an array.

Open to different name.. but don't have a good idea